### ISSUE TITLE
Ability to save model outcomes in tabular format CSV file?

### ISSUE THREAD

**Issue by pratikchhapolika** [huggingface/transformers#14972]
I am downloading the model https://huggingface.co/microsoft/Multilingual-MiniLM-L12-H384/tree/main microsoft/Multilingual-MiniLM-L12-H384 and then using it.

Transformer Version: '4.11.3'

I have written the below code:

model = tr.BertForSequenceClassification.from_pretrained("/home/pc/minilm_model",num_labels=2)
model.to(device)

print("hello")

training_args = tr.TrainingArguments(
    output_dir='/home/pc/proj/results2',          # output directory
    num_train_epochs=10,              # total number of training epochs
    per_device_train_batch_size=16,  # batch size per device during training
    per_device_eval_batch_size=32,   # batch size for evaluation
    learning_rate=2e-5,
    warmup_steps=1000,                # number of warmup steps for learning rate scheduler
    weight_decay=0.01,               # strength of weight decay
    logging_dir='./logs',            # directory for storing logs
    logging_steps=1000,
    evaluation_strategy="epoch",
    save_strategy="no"
)

print("hello")

trainer = tr.Trainer(
    model=model,                         # the instantiated 🤗 Transformers model to be trained
    args=training_args,                  # training arguments, defined above
    train_dataset=train_data,         # training dataset
    eval_dataset=val_data,             # evaluation dataset
    compute_metrics=compute_metrics
)

Is there way to retrieve my ( **I do not want to use tensor-board**) :

1. **Training loss** for every epoch
2. **Validation loss** for every epoch

In a tabular format ( May be in CSV file)

Then I want to plot my train and validation loss graph.

**Please help me what changes do I need to bring in the above code in case its feasible.**

**Comment by sid8491** [comment]
You need to add your own logging handler.
Check this: https://github.com/huggingface/transformers/issues/10454
